### ISSUE TITLE
Introduce threshold for sketches

### ISSUE THREAD

**Issue by seiflotfy** [skizzehq/skizze#114]
Each sketch should have a threshold map[string]int that will be populated until the limit its reached before replayign into the datastructure. This should allow us to ensure 100% accuracy for counts/cardinality/memberships under that treshold

**Comment by seiflotfy** [comment]
what it does is for example guarantee 100% accruacy under a threshold
its currently hardcoded to 10% of the maxunique elements
e.g: the bloom filter is set to take in max 1m unique values
up to 100k i am storing a dictionary
once the threshold is reached i replay into the bloomfilter
same thing for the count-min
however i dont think this is necessary for hyperloglog++
what do you guys think?
also I am thinking of getting rid of the 10% quota and make it adaptive
basically if i set up my bloom filter to take up to 1m values
its would be 1.44 MB
so i thought the threshold should be measured by “byte size"
so i use a dictionary until the dictionary is 1.44 MB big
then i replay into the bloom filter
what do you think?
